js/create_data.js

```diff
@@ -58,6 +58,33 @@ function Building(bldgName, stAddress, city, state, zip, country, purchasePrice,
     bldgDiversey.terminalCap = .065;
     bldgDiversey.bldgSize = 50000;
 }
+
+// ****
+// MJN example added:
```

Comment:
The "function Building" syntax you've used is useful if you need to check or validate data during the construction of your object - but that logic is only called if you pass the initial values to the function (see `bldgDiversey2` as an example).

If you do not have construction logic (you don't at the moment, but I suspect you might be planning some for the future), the other alternative is just to create a JavaScript object directly (see `bldgDiversey3`)

Reply:
That's right.   I plan on having construction logic in the future.   My initial goal was to create scaffolding that I'll build on later in the process.   Ideally, I should have folded tenants into building and probably will in the future.  With that said, I envision that building function will be used to create multiple buildings.

Reply:
That's correct.  In a future pass, I will fold Tenant object into building via an array.   Ultimately the goal will be create buildings via a constructor and store the building object on the server side data base.   But for now,  I just wanted to create the scaffolding for the future.